tests/detect-ldap-request-operation/README.md

```diff
@@ -0,0 +1,7 @@
+# Test Purpose
+
+Test keyword ldap.request.operation.
+
+## PCAP
+
+This PCAP was generated with flowsynth v1.4.1.
```

Comment:
I wonder if you can simply reuse some (not all) ldap tests and add a rules file with them